pyirf/benchmarks/energy_bias_resolution.py

```diff
@@ -106,3 +106,39 @@ def energy_bias_resolution(
         resolution_function
     )
     return result
+
+def energy_bias_resolution_from_energy_dispersion(
+    energy_dispersion,
+    migration_bins,
+):
+    """
+    Calculate bias and energy resolution.
+
+    Parameters
+    ----------
+    edisp:
+        Energy dispersion matrix of shape
+        (n_energy_bins, n_migra_bins, n_source_offset_bins)
+    migration_bins: numpy.ndarray
+        Bin edges for the relative energy migration (``reco_energy / true_energy``)
+    """
+
+    cdf = np.cumsum(energy_dispersion, axis=1)
+
+    n_energy_bins, _, n_fov_bins = energy_dispersion.shape
+
+    bias = np.zeros((n_energy_bins, n_fov_bins))
+    resolution = np.zeros((n_energy_bins, n_fov_bins))
+
+    for energy_bin in range(n_energy_bins):
```

Comment:
Unfortunately, I didn't find a nice way of vectorizing this

Reply:
not really a vectorization, but you could do at least the same as you do in the unit-test but using `itertools.product`

Reply:
That does not really change anything but remove one level of indentation. I have no strong preference for either and it doesn't change anything for run time.